### DISCUSSION TITLE
Multi file presentation

### DISCUSSION THREAD

**Discussion by dorich** [mdelobelle/metadatamenu#716] [Q&A]
When I create a multi file field that contains links to other files it always displays as a comma separated line of links. However in the setup for the field there is an option for "Indented List". But the display I have never appears in that form even though its selected in the field definition. 
Any suggestions as to what might be necessary to have the display be a list of links instead of the single line (Array)

Thanks

**Comment by LynetteCullens** [comment]
It should display as an indented list in source mode. That's what we're changing, the storage instead of the display.
